src/generated/resources/assets/gtceu/lang/en_us.json

```diff
@@ -3701,6 +3714,8 @@
   "gtceu.top.progress_computation": " / %s CWU",
   "gtceu.top.progress_sec": " / %s s",
   "gtceu.top.progress_tick": " / %s t",
+  "gtceu.top.recipe_output": "Recipe Outputs:",
+  "gtceu.top.stained": "Stained: %s",
```

Comment:
IMO this tooltip should be either "Dyed: %s" or "Colored: %s"

Reply:
renamed